awx/main/models/oauth.py

```diff
@@ -119,9 +118,6 @@ class Meta:
 
     def is_valid(self, scopes=None):
         valid = super(OAuth2AccessToken, self).is_valid(scopes)
-        if valid:
-            self.last_used = now()
```

Comment:
This happens *very* early in the request, and obtains an exclusive row lock until the transaction ends:

```python
  /venv/awx/lib64/python3.6/site-packages/django/core/handlers/base.py(113)_get_response()
  /usr/lib64/python3.6/contextlib.py(52)inner()
  /venv/awx/lib64/python3.6/site-packages/django/views/decorators/csrf.py(54)wrapped_view()
  /venv/awx/lib64/python3.6/site-packages/django/views/generic/base.py(71)view()
  /awx_devel/awx/api/generics.py(294)dispatch()
  /venv/awx/lib64/python3.6/site-packages/rest_framework/views.py(478)dispatch()
  /awx_devel/awx/api/generics.py(170)initialize_request()
  /venv/awx/lib64/python3.6/site-packages/rest_framework/request.py(223)user()
  /venv/awx/lib64/python3.6/site-packages/rest_framework/request.py(376)_authenticate()
  /awx_devel/awx
/api/authentication.py(46)authenticate()
  /venv/awx/lib64/python3.6/site-packages/oauth2_provider/contrib/rest_framework/authentication.py(29)authenticate()
  /venv/awx/lib64/python3.6/site-packages/oauth2_provider/oauth2_backends.py(169)verify_request()
  /venv/awx/lib64/python3.6/site-packages/oauthlib/oauth2/rfc6749/endpoints/base.py(87)wrapper()
  /venv/awx/lib64/python3.6/site-packages/oauthlib/oauth2/rfc6749/endpoints/resource.py(75)verify_request()
  /venv/awx/lib64/python3.6/site-packages/oauthlib/oauth2/rfc6749/tokens.py(351)validate_request()
  /venv/awx/lib64/python3.6/site-packages/oauth2_provider/oauth2_validators.py(377)validate_bearer_token()
> /awx_devel/awx/main/models/oauth.py(124)is_valid()
```